src/plugin.py

```diff
@@ -226,87 +236,86 @@ async def _handle_steam_guard_none(self) -> Authentication:
         if isinstance(authentication_data_or_error, ModelAuthPollResult):
             return await self._attempt_client_login(authentication_data_or_error)
         else:
-            logging.exception("Authentication poll failed despite the user not having 2FA. This is not recoverable.")
+            logger.exception("Authentication poll failed despite the user not having 2FA. This is not recoverable.")
             raise UnknownBackendResponse()
 
     async def _attempt_client_login(self, poll_result: ModelAuthPollResult) -> Authentication:
         auth = await self._attempt_client_login_common(self._unauthed_steam_id, poll_result.account_name, poll_result.refresh_token)
-        if (auth is None):
+        if auth is None:
             logger.warning("Client Login failed despite credential login succeeding. Nothing to fall back to.")
             raise UnknownBackendResponse()
         else:
             return auth
 
-    async def _attempt_client_login_common(self, steam_id: int, account_name:str, refresh_token: str) -> Optional[Authentication]:
+    async def _attempt_client_login_common(self, steam_id: int, account_name: str, refresh_token: str) -> Optional[Authentication]:
         maybe_auth_data = await self._model.steam_client_login(account_name, steam_id, refresh_token, get_os())
-        if (maybe_auth_data is None):
+        if maybe_auth_data is None:
             return None
         else:
             return Authentication(str(steam_id), account_name)
 
     def tick(self):
         self._model.tick()
-        pass
 
     async def shutdown(self):
-        """Called when GOG Galaxy Client is shutdown or the plugin is disconnected by the user. 
-        """
+        """Called when GOG Galaxy Client is shutdown or the plugin is disconnected by the user."""
```

Comment:
IIRC pep 8 wants the end `"""` for docstrings on their own line, but personally idc. Not an issue.

Reply:
flake didn't report an issue for any variant so I went with the one that doesn't hurt my eyes :'D